shell.nix

```diff
@@ -10,6 +10,7 @@ stdenv.mkDerivation {
     git-crypt
     nixops
     sqlite
+    jq
```

Comment:
I want to get rid of the `shell.nix`. It would be better to start the bootstrap script with a nix shell shebang with only the packages the script needs.